docs/launch.md

```diff
@@ -115,3 +141,45 @@ fs_1.default.rmSync is not a function
 ```
 
 **Solution**. Make sure that the version of `node.js` installed on your computer is `14.14.0` or higher.
+
+### Rust compilation problems
+
+**Problem**. Compilation problems with lexical-core.
+
+**Solution**. In `zksync/Cargo.toml` append at the end:
+
+- `lexical-core = {git = 'https://github.com/Gelbpunkt/rust-lexical', branch = 'fix-warnings-and-update-deps'}`
+
+Then run zk init again. The suggestion from discord, to try `cargo update lexical-core` does not work
+
```

Comment:
```suggestion

**Problem**. Compilation problems with `bynarien` due to `Command Line Tools for Xcode v13.2+`.

Error:
```
In file included from <project_root>/sdk/binaryen/src/wasm/wasm-type.cpp:28:
<project_root>/sdk/binaryen/src/wasm-type.h:365:10: error: definition of implicit copy constructor for 'Tuple' is deprecated because it has a user-declared copy assignment operator [-Werror,-Wdeprecated-copy]
  Tuple& operator=(const Tuple&) = delete;
         ^
<project_root>/sdk/binaryen/src/wasm/wasm-type.cpp:51:51: note: in implicit copy constructor for 'wasm::Tuple' first required here
  TypeInfo(const Tuple& tuple) : kind(TupleKind), tuple(tuple) {}
                                                  ^
1 error generated.
make[2]: *** [src/wasm/CMakeFiles/wasm.dir/wasm-type.cpp.o] Error 1
make[1]: *** [src/wasm/CMakeFiles/wasm.dir/all] Error 2
make: *** [all] Error 2
error Command failed with exit code 2.
```

```bash
❯ clang++ --version
Apple clang version 13.1.6 (clang-1316.0.21.2)
Target: x86_64-apple-darwin21.4.0
```

**Solution**. Download the latest working `Command Line Tools for Xcode` from [developer.apple.com](https://developer.apple.com/download/all/?q=command%20line%20tools). An apple account is required.

```